src/components/ApiErrors/index.tsx

```diff
@@ -52,9 +52,20 @@ const ApiErrors = React.forwardRef<HTMLDivElement, ApiErrorsProps>(
       )[2];
     } else if (lang !== "en" && typeof problemDetail !== "string") {
       const errorToTranslate = problemDetail?.detail;
```

Comment:
I think we can remove the nullcheck on problemDetail here cuz it's already been performed on L30 above.   (this only matters because I was trying to reduce new variables in comment below)